### DISCUSSION TITLE
GPU with 3 fans, but only shows 2 in FanControl

### DISCUSSION THREAD

**Discussion by BaconismTV** [Rem0o/FanControl.Releases#752] [Q&A] [answered]
I have the ZOTAC RTX 3070Ti Amp Extreme Holo, and it has 3 fans, but only 2 show up in FanControl, is this intended?  And if it is, does 1 of the GPU fan controls control 2 of the fans then?

**Comment by xhul-dev** [comment, marked as answer]
> does 1 of the GPU fan controls control 2 of the fans then?

Yes, you're not the first one to report that.
It's probably cheaper for the manufacturer.

**Reply by Rem0o** [reply]
These cards have 2 channels. 2 fans are wired to 1 channel, the other fan is wired to the other.
